peps/pep-0750.rst

```diff
@@ -1228,41 +1279,58 @@ The Final Home for ``Template`` and ``Interpolation``
 
 Previous versions of this PEP proposed placing the ``Template`` and
 ``Interpolation`` types in: ``types``, ``collections``, ``collections.abc``,
-and even in a new top-level module, ``templatelib``. As of this writing, no core
-team consensus has emerged on the final location for these types. The current
-PEP leaves this open for a final decision.
-
-One argument in favor of a new top-level ``templatelib`` module is that it would
-allow for future addition of related methods (like ``convert()``) and for
-potential future template processing code to be added to submodules
-(``templatelib.shell``, etc.).
+and even in a new top-level module, ``templatelib``. The final decision was to
+place them in ``string.templatelib``.
 
 
 Enable Full Reconstruction of Original Template Literal
 -------------------------------------------------------
 
 Earlier versions of this PEP attempted to make it possible to fully reconstruct
 the text of the original template string from a ``Template`` instance. This was
-rejected as being overly complex.
+rejected as being overly complex. The mapping between template literal source
+and the underlying AST is not one-to-one and there are several limitations with
```

Comment:
If this is the first instance of "AST", could we spell out "abstract syntax tree (AST)"?